File: Medium/JavaScript/1641. Count Sorted Vowel Strings.js
var countVowelStrings = function(n) {
    let a = e = i = o = u = 1;

    for(let j = 1; j < n; j++){
        a += e + i + o + u;
        e += i + o + u;
        i += o + u;
        o += u;
        // u doesn't change
    }

    return a + e + i + o + u;
};

File: Medium/Python/1641. Count Sorted Vowel Strings.py
class Solution(object):
    def countVowelStrings(self, n):
        a = e = i = o = u = 1

        for j in range(1, n):
            a += e + i + o + u
            e += i + o + u
            i += o + u
            o += u
            # u doesn't change

        return a + e + i + o + u
